site/testsuites/testsuite-javascript/__tests__/tbdex/pfi/orders.js

```diff
@@ -0,0 +1,59 @@
+import { Order, OrderStatus, Close } from '@tbdex/http-server'
+import { DidDhtMethod } from '@web5/dids';
+
+var pfiDid = await DidDhtMethod.create({
+    publish: false,
+    services: [{
+        id: 'pfi',
+        type: 'PFI',
+        serviceEndpoint: 'tbdex-pfi.tbddev.org'
+    }]
+})
+
+var senderDid = await DidDhtMethod.create({ publish: false })
+
+var config = {
+    did: pfiDid    
+}
+
+var dataProvider = {
+    write: (args) => {
+        // do nothing
+    }
+}
+
+var message = Order.create({
```

Comment:
can you rename this to `orderMessage`. I had a hard time keeping up with which one this was for as this guide has 3 different types of messages: Order, OrderStatus, and Close

```suggestion
var orderMessage = Order.create({
```